redash/query_runner/druid.py

```diff
@@ -0,0 +1,93 @@
+try:
+    from pydruid.db import connect
+    enabled = True
+except ImportError:
+    enabled = False
+
+from redash.query_runner import BaseQueryRunner, register
+from redash.utils import json_dumps, json_loads
+
+TYPES_MAP = {
+}
```

Comment:
![image](https://user-images.githubusercontent.com/71468/48131665-298c5c80-e291-11e8-9c6d-0f0a29e678f9.png)

?

Reply:
As far as I understand, Druid supports only one custom data type - `OTHER/COMPLEX`; however, I'm not sure how to represent it in Redash

Reply:
The types map is not only for custom stuff.